resources/files/ruby_219/rbconfig/rbconfig-powerpc64le-linux-gnu.rb

```diff
@@ -196,8 +196,8 @@ module RbConfig
   CONFIG["build_vendor"] = "ibm"
   CONFIG["build_cpu"] = "powerpc64le"
   CONFIG["build"] = "powerpc64le-ibm-linux-gnu"
-  CONFIG["RUBY_RELEASE_DATE"] = "2016-03-30"
-  CONFIG["RUBY_PROGRAM_VERSION"] = "2.3.3"
+  CONFIG["RUBY_RELEASE_DATE"] = "2015-04-13"
```

Comment:
I'm pretty sure this isn't relevant to fixing the error we were seeing, I did it to make things consistent with the other rbconfigs for this version.